app/subscriber/src/components/layout/DefaultLayout.tsx

```diff
@@ -52,10 +51,10 @@ export const DefaultLayout: React.FC<ILayoutProps> = ({ children, ...rest }) =>
 
   // check if width is mobile and if so collapse the sidebar or when window is being resized
   React.useEffect(() => {
-    if (width && width < 768 && !collapsed && !advancedSearch) {
+    if (width && width < 768 && !collapsed) {
```

Comment:
Why can't we use css media query?

Reply:
adjusting!

Reply:
oh wait, it's because I need to call the hook to collapse the sidebar dynamically when the screen reaches a certain size so I needed to check width here instead of the scss